src/components/common/layout/header/Header.tsx

```diff
@@ -1,36 +1,10 @@
-import React, { useState } from 'react';
-import {
-  AcademicCapIcon,
-  BriefcaseIcon,
-  ClipboardIcon,
-  HomeIcon,
-  LockClosedIcon,
-  UsersIcon,
-} from '@heroicons/react/24/outline';
-import Link from 'next/link';
+import React from 'react';
+import { useMediaQuery } from '@mui/material';
 
-import Button, {
-  ButtonSize,
-  ButtonVariant,
-} from '@/components/common/ui/button';
 import useAuthentication from '@/hooks/use-authentication';
-import useIsMobile from '@/hooks/use-is-mobile/UseIsMobile';
 
-import { BurgerMenu } from '../../custom-svg/BurgerMenu';
-import {
-  IconButton,
-  IconButtonColor,
-  IconButtonSize,
-} from '../../ui/icon-button/IconButton';
-import { CloseButton } from '../../ui/icon-button/variants';
-import { TabItem, TabItemContentPosition } from '../../ui/tab';
-import { TabItemContentSize } from '../../ui/tab/tab-item/TabItem';
-
-import { HeaderDesktopCard } from './components/header-desktop-card';
-import HeaderDivider from './components/header-divider/HeaderDivider';
-import { HeaderMobileCard } from './components/header-mobile-card/HeaderMobileCard';
-
-import styles from './Header.module.scss';
+import DesktopHeader from './components/desktop-header/DesktopHeader';
+import MobileHeader from './components/mobile-header/MobileHeader';
 
 const roleMapper = {
```

Comment:
Make another folder called 'utils' and make function transformData. Move roleMapper and all of the user data tranformation there

Reply:
If we squash mobile and desktop headers into one component, utils don`t needed